partner_financial_risk/views/res_partner_view.xml

```diff
@@ -56,5 +56,13 @@
         </page>
     </field>
 </record>
+    <record id="view_partner_property_form" model="ir.ui.view">
```

Comment:
Hi @pedrobaeza, you mean leave the credit_limit field in the account view as it is or I should add the new attributes of the field?